packages/monaco-editor-wrapper/src/editorAppBase.ts

```diff
@@ -223,3 +227,56 @@ export abstract class EditorAppBase {
 export const isVscodeApiEditorApp = (wrapperConfig: WrapperConfig) => {
     return wrapperConfig.editorAppConfig?.$type === 'vscodeApi';
 };
+
+export const isCodeUpdateRequired = (config: EditorAppBaseConfig, modelUpdate: ModelUpdate) => {
+    let updateRequired = modelUpdate.code !== undefined && modelUpdate.code !== config.code;
+    updateRequired = updateRequired || modelUpdate.codeOriginal !== config.codeOriginal;
```

Comment:
Same as below, but small enough here that's it's probably fine.

Reply:
Condensed the code a bit. ⬇️